sdk/core/src/services/WalletStorageService.ts

```diff
@@ -1,6 +1,7 @@
 import { toRpcSig, ecsign } from 'ethereumjs-util'
```

Comment:
we can convert this service into 3 different classes:
- BloomVaultService: old bloom vault related auth and crud code
- AffinidiVaultService: new affinidi vault related auth, hashing, and crud code
- WalletStorageService: combine BloomVault + AffinidiVault

that should make the code more readable, make it easier to unit test, review 3 different classes.

Reply:
@inga-affinidi regarding [your comment](https://github.com/affinityproject/affinidi-core-sdk/pull/117#issuecomment-883229623), I think we are talking about similar thing. But I think separating to 3 classes could be better. Because both Affinidi and Bloom has different characteristics with their own auth logic and Affinidi Vault having hashing logic. 

WalletStorageService could be the adapter that uses these two services to implement reading from both, merging the results etc. kind of logic.

Reply:
@Yengas  Agree, that's also an option.

Reply:
After combining all the logic in WalletStorageService from both vaults looks like will be better to separate them out. I thought about it initially, but was hesitating that we will have pretty long dependency chain of services.